app/views/projects/_index_dashboard.html.erb

```diff
@@ -2,144 +2,200 @@
 <link rel="stylesheet" type="text/css" href="https://webfonts.artsy.net/unica-webfonts.css" />
 <style type="text/css">
 body {
-  font-family: "Unica77LLWebRegular"
+  font-family: "Unica77LLWebRegular";
+  font-size: 14px;
 }
-svg {
-  padding-top: 60px;
-  padding-left: 60px;
+
+h1 {
+  font-size: 18px;
+  color: #000000;
+  letter-spacing: 0;
+  line-height: 24px;
   clear: both;
+  margin: 30px;
+  font-family: "Unica77LLWebMedium";
+  font-weight: normal;
 }
 
 h2 {
   font-size: 20px;
-  font-weight: bold;
+  font-family: "Unica77LLWebMedium";
+  font-weight: normal;
   color: black;
 }
 
-.released {
-  padding-left: 48px;
+p {
+  margin-top: -4px;
+  font-size: 14px;
+  height: 18px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow-y: hidden;
 }
-.released h2 {
+
+.todo {
+  clear: both;
+}
+
+.todo .project {
+  border: 1px solid #E5E5E5;
+  border-radius: 2px;
+  padding: 16px;
+  width: calc(286px - 32px);
+  height: calc(269px - 32px);
+  margin: 30px;
+  margin-top: 0;
   float: left;
-  margin-left: 32px;
 }
 
-.green-circle {
-  border-radius: 50%;
-  width: 10px;
-  height: 10px; 
-  background-color: #35D750;
-  display: inline-block;
+.todo .project.red {
+  border: 1px solid #F7625A;
+}
+
+.todo .project h2 {
+  font-size: 28px;
+  margin: 0;
+  padding: 0;
+}
+
+.todo .project .comparison {
+  margin-bottom: 0px;
   margin-right: 16px;
+  min-height: 40px;
+  background: url(/assets/arrow-d9bccef77260cf5af9399ae03a67b3e0f78b0ab67a81224e6dab827ba2ffbe4c.svg) 0.5px 20px no-repeat;
```

Comment:
This should use the built-in helpers rather than hard-code an asset digest. In this context I think this would work:

    background: url(<%= image_url('arrow.svg') %>) 0.5px 20px no-repeat;

But better would be to move these to `projects.scss` where they can use things like `image-url(...)` inline.